docs/ldml/tr35-numbers.md

```diff
@@ -578,6 +578,20 @@ If there is an explicit negative subpattern, it serves only to specify the negat
 
 > **Note:** The thousands separator and decimal separator in patterns are always ASCII ',' and '.'. They are substituted by the code with the correct local values according to other fields in CLDR. The same is true of the - (ASCII minus sign) and other special characters listed above.
 
+A currency decimal pattern normally contains a currency symbol placeholder (one to four ¤ characters). The currency symbol placeholder may occur before the first digit, after the last digit symbol, or where the decimal symbol would otherwise be placed.
+
+Before | After | Decimal
+-------|-------|-------
+¤#,##0.00|#,##0.00¤|#,##0¤00
```

Comment:
Can the pattern `#,##0¤00` ever actually occur in CLDR data?  If it can, then we'll need to change the EBND for number patterns (assuming there is one).

Reply:
I thought we had an EBNF for these, but I can't find one. These number patterns continue becoming more and more complex to parse. I'm going to need to implement a parser in ICU4X. Before I do that, I would really like to have a formal grammar that I can implement.

Reply:
/[0#]¤[0#]/ does not occur in any CLDR data (current or past), so I don't think there is any real compatibility concern.

I think an EBNF is an excellent idea. Please file a ticket for that and we can work on it (for next release).